File: app/src/main/java/nyc/c4q/hakeemsackes_bramble/infinity_loop_clone/tile_database/DB_GameLayout.java
package nyc.c4q.hakeemsackes_bramble.infinity_loop_clone.tile_database;

/**
 * Created by hakeemsackes-bramble on 11/30/17.
 */

public class DB_GameLayout {

    private String mGameLayout;
    private String mGameColor;

    public DB_GameLayout() {
    }

    public DB_GameLayout(String gameLayout) {
        this.mGameLayout = gameLayout;
    }

    public String getmGameLayout() {
        return mGameLayout;
    }

    public void setmGameLayout(String mGameLayout) {
        this.mGameLayout = mGameLayout;
    }

    public String getmGameColor() {
        return mGameColor;
    }

    public void setmGameColor(String mGameColor) {
        this.mGameColor = mGameColor;
    }
}

File: app/src/main/java/nyc/c4q/hakeemsackes_bramble/infinity_loop_clone/tile_database/DB_GameLayout.kt
package nyc.c4q.hakeemsackes_bramble.infinity_loop_clone.tile_database

/**
 * Created by hakeemsackes-bramble on 11/30/17.
 */
class DB_GameLayout {
    private var mGameLayout: String? = null
    private var mGameColor: String? = null

    constructor()
    constructor(gameLayout: String?) {
        mGameLayout = gameLayout
    }

    fun getmGameLayout(): String? {
        return mGameLayout
    }

    fun setmGameLayout(mGameLayout: String?) {
        this.mGameLayout = mGameLayout
    }

    fun getmGameColor(): String? {
        return mGameColor
    }

    fun setmGameColor(mGameColor: String?) {
        this.mGameColor = mGameColor
    }
}